go/api/commands.go

```diff
@@ -150,8 +231,123 @@ type StringCommands interface {
 	//  The value of key after the decrement.
 	//
 	// For example:
-	//  result, err := client.Decr("key");
+	//  key: 1
+	//  result, err := client.DecrBy("key", 2);
+	//  result: -1
 	//
 	// [valkey.io]: https://valkey.io/commands/decrby/
 	DecrBy(key string, amount int64) (int64, error)
+
+	// Returns the length of the string value stored at key.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key - The key to check its length.
+	//
+	// Return value:
+	//  The length of the string value stored at key.
+	//  If key does not exist, it is treated as an empty string, and the command returns 0.
+	//
+	// For example:
+	//  key: value
+	//  result, err := client.Strlen("key");
+	//  result: 5
+	//
+	// [valkey.io]: https://valkey.io/commands/strlen/
+	Strlen(key string) (int64, error)
+
+	// Overwrites part of the string stored at key, starting at the specified offset, for the entire length of value.
```

Comment:
The offset is byte's offset, please mention it.
Some char's take more than one byte, we want to make sure the user don't get confused.

Reply:
Modified.